### PR TITLE
add override to fix collision with jupyter-core

### PR DESCRIPTION
**Pull request by vale981**

### CODE REVIEW

File: overrides.nix [hunk 1]
```diff
@@ -633,6 +633,12 @@ self: super:
       # dependencies. It does however have a jupyter.py file that conflicts
       # with jupyter-core so this meta solves this conflict.
       meta.priority = 100;
+      
+      # remove, what is already present in jupyter-core
+      postInstall = ''
+        rm -f $out/lib/python*/site-packages/__pycache__/jupyter.cpython-39.pyc
```
Comment (cpcloud) [inline overrides.nix:639]:
```suggestion
        rm -f $out/${super.sitePackages}/__pycache__/jupyter.cpython-*.pyc
```

File: overrides.nix [hunk 2]
```diff
@@ -641,6 +647,15 @@ self: super:
       buildInputs = (old.buildInputs or [ ]) ++ [ self.jupyter-packaging ];
     }
   );
+  
+  jupyter-core = super.jupyter-core.overridePythonAttrs (
+    old: {
+      postInstall = ''
+        rm -f $out/lib/python*/site-packages/__pycache__/jupyter.cpython-39.pyc
```
Comment (cpcloud) [inline overrides.nix:654]:
```suggestion
        rm -f $out/${super.sitePackages}/__pycache__/jupyter.cpython-*.pyc
```

File: overrides.nix [hunk 3]
```diff
@@ -633,6 +633,12 @@ self: super:
       # dependencies. It does however have a jupyter.py file that conflicts
       # with jupyter-core so this meta solves this conflict.
       meta.priority = 100;
+      
+      # remove, what is already present in jupyter-core
+      postInstall = ''
+        rm -f $out/lib/python*/site-packages/__pycache__/jupyter.cpython-39.pyc
+        rm -f $out/lib/python*/site-packages/jupyter.py
```
Comment (cpcloud) [inline overrides.nix:640]:
```suggestion
        rm -f $out/${super.sitePackages}/jupyter.py
```

File: overrides.nix [hunk 4]
```diff
@@ -641,6 +647,15 @@ self: super:
       buildInputs = (old.buildInputs or [ ]) ++ [ self.jupyter-packaging ];
     }
   );
+  
+  jupyter-core = super.jupyter-core.overridePythonAttrs (
+    old: {
+      postInstall = ''
+        rm -f $out/lib/python*/site-packages/__pycache__/jupyter.cpython-39.pyc
+        rm -f $out/lib/python*/site-packages/jupyter.py
```
Comment (cpcloud) [inline overrides.nix:655]:
```suggestion
        rm -f $out/${super.sitePackages}/jupyter.py
```

**Review by cpcloud — CHANGES_REQUESTED** [review]

**Comment by adisbladis** [comment]
I don't even see why this would be needed as we do set `meta.priority` in this case?

**Comment by adisbladis** [comment]
Feel free to reopen once the problems have been fixed.

**Pull request closed without merging**